odoo/http.py

```diff
@@ -1201,7 +1204,7 @@ def _inject_future_response(self, response):
         response.headers.extend(self.future_response.headers)
         return response
 
-    def make_response(self, data, headers=None, cookies=None):
+    def make_response(self, data, headers=None, cookies=None, status=200):
```

Comment:
Making all the parameters with default keyword-only could be a good idea no?

Reply:
I don't like the order either, the problem is that they are call sites here and there that pass the header argument as positional argument. IMO it is a duty for another PR if you agree.

Reply:
Sure, works for me. was more of a "could be a nice opportunity to put keyword parameters everywhere".

Plus soon we'll be able to use posonly parameters as well
![heresy](https://user-images.githubusercontent.com/7571158/165502027-00d3c0e7-ca97-4dcb-b0be-edef96d720c1.jpg)

Reply:
A quick grep revealed that it is quite some work, including the places where we use `make_response` to craft a JSON response that could benefit from `make_json_response`. Well, we both know where it is going...